libraries/commerce/product/actions/fetchProducts.js

```diff
@@ -72,8 +74,16 @@ const fetchProducts = ({
 
     const { sort = DEFAULT_SORT } = hashParams;
 
+    let getProductsRequestParams;
+    if (pipeline === pipelines.SHOPGATE_CATALOG_GET_PRODUCTS) {
+      getProductsRequestParams = configuration.get(DEFAULT_PRODUCTS_FETCH_PARAMS);
```

Comment:
Like `getProducts` the `getProduct` pipeline only adds the characteristics property when the parameter is sent with the input parameters. To be able to see characteristics on favourite list items which where added via a child product on the PDP, you also need to add the parameter there.